api/envoy/api/v2/core/config_source.proto

```diff
@@ -29,6 +29,13 @@ message ApiConfigSource {
     REST = 1;
     // gRPC v2 API.
     GRPC = 2;
+    // Using the incremental xDS gRPC service, i.e. IncrementalDiscovery{Request,Response}
+    // rather than Discovery{Request,Response}. Rather than sending Envoy the entire state
+    // with every update, the xDS server only sends what has changed since the last update.
+    //
+    // INCREMENTAL_GRPC IS NOT YET ENTIRELY IMPLEMENTED! Initially, only CDS is available.
```

Comment:
Can you use RST warning markup, e.g. `.. WARNING::` here? I would suggest the protodoc experimental/value annotation (https://github.com/envoyproxy/envoy/blob/74ef1a9030c0179412a6938439896a18cc81bf74/tools/protodoc/protodoc.py#L90), but this would apply to the full enum, whereas the experimental/draft bit is just this value (unless you want to hack protodoc?).

Reply:
I've added the RST warning markup. I'm not familiar with protodoc; it sounds like it would be something else that would then generate this markup? If it would inappropriately mark the whole enum, is it ok to just keep it manual like this?

Reply:
Sure